src/app/helptext/sharing/iscsi/iscsi-wizard.ts

```diff
@@ -50,7 +50,9 @@ export default {
  Mutual CHAP."),
 
     ip_placeholder: T("IP"),
-    ip_tooltip: T("Select the IP address associated with an interface or the wildcard address of 0.0.0.0 (any interface)."),
+    ip_tooltip: T("Select the IP addresses associated with an interface \
+ or the wildcard addresses 0.0.0.0 (all IPv4 addresses) and/or :: (all \
```

Comment:
Don't use "and/or".  Also, check with Lola on whether 0.0.0.0 and :: can be combined.  I didn't think they could, but there was another value for "everything".  Also, "associated with an interface" is kind of misleading.  With iSCSI, this is to choose allowed or denied IP addresses, or which address the server daemon listens on.